.gitignore

```diff
@@ -1,4 +1,7 @@
 install_test
+Pipfile
```

Comment:
Could you delete the changes in .gitignore, please? These files are not included in this project.

Reply:
Done ✅